core/start-blueos-core

```diff
@@ -15,6 +15,7 @@ MAV_COMPONENT_ID_ONBOARD_COMPUTER4=194
 # Enable Rust backtrace for all programs
 RUST_BACKTRACE=1
 
+RUN_AS_BLUEOS_USER="sudo -u blueos"
```

Comment:
in case `sudo` is not available, you can `su blueos -c "commandline with arguments"`

Reply:
the quotes in there make commands insanely hard to assemble. Do you want to take a look at it? I tried for a while but ended up giving up

Reply:
it should be something like:
```Bash

BLUEOS_USER='blueos'

run_as_blueos_user () {
  su "$BLUEOS_USER" -c "$1"
}

run_as_blueos_user 'ls -lah /dev/video*'
```

Reply:
didn't quite work, I think this need to be on its own file so it can be used inside the new tmux sessions.
I'll try it in a subsequent pr